YoutubeExplodeTests/YoutubeClientPlaylistParsingTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using YoutubeExplode.Services;
+
+namespace YoutubeExplode.Tests
+{
+    [TestClass]
+    public class YoutubeClientPlaylistParsingTests
```

Comment:
Yeah, I also try to use videos that are unlikely to change, like music videos.
Indeed, with unit tests you don't have to worry about that, but you do have to worry about updating the `PlaylistpageX.json` when youtube changes something. If you find a playlist that is very unlikely to change you nearly eliminate the first problem and don't produce the second.
I initially tried to mock stuff for VideoInfo as well, but quickly realized that there's no point in regards to a project like this one. Keeping both unit and integration tests doesn't offer much advantage over having just the integration tests, but takes a considerable amount of effort to maintain.